packages/common/src/networks/data.json

```diff
@@ -45,6 +68,76 @@
     "multicall2Address": "0x275617327c958bD06b5D6b871E7f491D76113dd8",
     "multicall3Address": "0xcA11bde05977b3631167028862bE2a173976CA11"
   },
+  {
+    "id": "zksync",
+    "chainId": 324,
+    "name": "zkSync",
+    "explorerUrl": "https://explorer.zksync.io/",
+    "rpcUrl": "https://mainnet.era.zksync.io/",
+    "nativeToken": {
+      "chainId": 324,
+      "address": "0x0000000000000000000000000000000000000000",
+      "decimals": 18,
+      "symbol": "ETH",
+      "name": "Ethereum"
+    },
+    "wrappedNativeToken": {
+      "chainId": 324,
+      "address": "0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91",
+      "decimals": 18,
+      "symbol": "WETH",
+      "name": "Wrapped Ether"
+    },
+    "multicall2Address": "0x1fCD733705ec1CF534dD19ecF627F013352e776F",
+    "multicall3Address": "0xF9cda624FBC7e059355ce98a31693d299FACd963"
+  },
+  {
+    "id": "metis",
+    "chainId": 1088,
+    "name": "Metis",
+    "explorerUrl": "https://andromeda-explorer.metis.io/",
+    "rpcUrl": "https://metis-mainnet.public.blastapi.io",
+    "nativeToken": {
+      "chainId": 1088,
+      "address": "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
+      "decimals": 18,
+      "symbol": "METIS",
+      "name": "Metis"
+    },
+    "wrappedNativeToken": {
+      "chainId": 1088,
+      "address": "0x71802e8F394BB9d05a1b8E9d0562917609FD7325",
+      "decimals": 18,
+      "symbol": "WMETIS",
+      "name": "Wrapped METIS"
+    },
+
+    "multicall2Address": "0xc39aBB6c4451089dE48Cffb013c39d3110530e5C",
+    "multicall3Address": "0xcA11bde05977b3631167028862bE2a173976CA11"
+  },
+  {
+    "id": "base",
+    "chainId": 8453,
+    "name": "Base",
+    "explorerUrl": "https://basescan.org/",
+    "rpcUrl": "https://rpc.ankr.com/base",
+    "nativeToken": {
+      "chainId": 8453,
+      "address": "0x0000000000000000000000000000000000000000",
+      "decimals": 18,
+      "symbol": "ETH",
+      "name": "Ethereum"
+    },
+    "wrappedNativeToken": {
+      "chainId": 8453,
+      "address": "0x4200000000000000000000000000000000000006",
+      "decimals": 18,
+      "symbol": "WETH",
+      "name": "Wrapped Ether"
+    },
+    "multicall2Address": "0x091e99cb1C49331a94dD62755D168E941AbD0693",
```

Comment:
That's pretty weird since the Uniswap team put this address in the [Multicall2 address column](https://docs.uniswap.org/contracts/v3/reference/deployments). Will use the multicall3Address for the multicall2Address.

Reply:
It seems like Uniswap has already transitioned to using its own Multicall. I think I should find time to refine how we retrieve quotes from their interface.

Reply:
fixed